lib/_stream_readable.js

```diff
@@ -325,7 +325,7 @@ Readable.prototype.setEncoding = function(enc) {
 // Don't raise the hwm > 8MB
 const MAX_HWM = 0x800000;
 function computeNewHighWaterMark(n) {
-  if (n >= MAX_HWM) {
+  if (n > MAX_HWM) {
```

Comment:
This causes the else branch to be taken when n is MAX_HWM. That seems less efficient and possibly a bug?

Reply:
OK, Thanks I've removed that modification.